CONTRIBUTING.md

```diff
@@ -1,3 +1,40 @@
 # Contributing
 
 TODO
```

Comment:
What's the TODO for?
Also, should we add something about testing in here? We don't want to generate the icon components if the tests fail.

Reply:
Great comment about the testing, thank you i'll add it

Reply:
The TODO  was for the rest of the conitrbuting document. How to dev in this repo and everything other than icons